src/main/java/com/denizenscript/depenizen/bukkit/objects/superiorskyblock/SuperiorSkyblockIslandTag.java

```diff
@@ -0,0 +1,397 @@
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.banned_players>
+        // @returns ListTag(PlayerTag)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns a list of players banned on the specified island.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "banned_players", (attribute, object) -> {
+            return new ListTag(object.getIsland().getBannedPlayers(), players -> new PlayerTag(players.asPlayer()));
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.bonus_level>
+        // @returns ElementTag(Decimal)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the bonus level of an island.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "bonus_level", (attribute, object) -> {
+            return new ElementTag(object.getIsland().getBonusLevel());
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.bonus_worth>
+        // @returns ElementTag(Decimal)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the bonus worth of an island.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "bonus_worth", (attribute, object) -> {
+            return new ElementTag(object.getIsland().getBonusWorth());
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.coop_members>
+        // @returns ListTag(PlayerTag)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the coop members of the specified island, if any.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "coop_members", (attribute, object) -> {
+            return new ListTag(object.getIsland().getCoopPlayers(), players -> new PlayerTag(players.asPlayer()));
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.current_visitors>
+        // @returns ListTag(PlayerTag)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the players currently within the bounds of the specified island.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "current_visitors", (attribute, object) -> {
+            return new ListTag(object.getIsland().getAllPlayersInside(), players -> new PlayerTag(players.asPlayer()));
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.description>
+        // @returns ElementTag
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the description of the specified island, if any.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "description", (attribute, object) -> {
+            return object.getIsland().getDescription().isBlank() ? null : new ElementTag(object.getIsland().getDescription(), true);
```

Comment:
A. any specific reason for `#isBlank` over `#isEmpty`?
B. Are we sure this should be nullable as opposed to matching what the plugin does and returning an empty element?